core/src/cluster_info.rs

```diff
@@ -95,6 +96,9 @@ const MAX_GOSSIP_TRAFFIC: usize = 128_000_000 / PACKET_DATA_SIZE;
 
 /// Keep the number of snapshot hashes a node publishes under MAX_PROTOCOL_PAYLOAD_SIZE
 pub const MAX_SNAPSHOT_HASHES: usize = 16;
+const GOSSIP_PING_TOKEN_SIZE: usize = 32;
+const GOSSIP_PING_CACHE_CAPACITY: usize = 16384;
```

Comment:
Should this be based on the number of nodes in gossip instead of a constant?

Reply:
@behzadnouri, got it, thanks! I think the most convenient available way would be to use `cluster_info.gossip_peers()`. It's not the most efficient because it scans the table (broadcast does something similar with `cluster_info.tvu_peers()`) but may not be too bad if done periodically.

Reply:
thanks, since the alternatives would require table scan and locking, i lean toward keeping this constant for now, and later test alternatives.